### DISCUSSION TITLE
telegram version

### DISCUSSION THREAD

**Discussion by ytpk** [drizzle-mizzle/CharacterAI-Discord-Bot#11] [Ideas]
is it possible to do the same but for telegram plsplspls?

**Comment by drizzle-mizzle** [comment]
Of course it is, but I really doubt that there's anyone who's gonna use it...

**Reply by ytpk** [reply]
I really want to, but I saw the last similar project in December, and cloudflare protection broke it

**Reply by drizzle-mizzle** [reply]
Well, won't promise, but maybe I'll make one later.
